test/reconstruction/enzyme_constrained.jl

```diff
@@ -43,7 +43,7 @@
             lower_bound = [-1000.0, -1.0],
             upper_bound = [nothing, 12.0],
         ) |>
-        with_enzyme_constraints(total_gene_product_mass)
+        with_enzyme_constraints(total_enzyme_capacity = total_gene_product_mass)
```

Comment:
"capacity" maybe just call it "total_enzyme_mass" to totally explain that it's the mass

Reply:
Hmm I would say capacity is better here, because its a ratio of total enzyme mass to total cellular mass